yacc/reader.c

```diff
@@ -904,14 +904,15 @@ void declare_tokens(int assoc)
     if (c == EOF) unexpected_EOF();
     if (c == '<')
     {
+        column = cptr - line;
         tag = get_tag();
         c = nextc();
         if (c == EOF) unexpected_EOF();
     }
 
     for (;;)
     {
-        if (isalpha(c) || c == '_' || c == '.' || c == '$')
+        if (In_bitmap(caml_ident_start, c) || c == '.' || c == '$')
```

Comment:
You will have to fix `IS_IDENT` (used in `get_name`) too. As it is, you're moving from locale-dependent behavior to a locale-dependent inconsistency that (if I read the code correctly) leads to an infinite loop.

Reply:
An, good point.

Reply:
Another point: we are trying to move OCaml toward Unicode, so it's probably not a good idea to push ocamlyacc toward better handling of Latin-1...